leftwm/src/utils/cacher.rs

```diff
@@ -0,0 +1,58 @@
+use std::{fs::File, path::PathBuf};
+use xdg::{BaseDirectories, BaseDirectoriesError};
+
+lazy_static! {
```

Comment:
Is there possibly a way to avoid introducing this new dependency?

Reply:
Hm... I thought that having a singleton would be the best since it would be kinda weird to have multiple "cachers". So the answer is actually no sadly.... but I'll ask my friends if they've got any ideas.